src/scss/skin-modern/_skin.scss

```diff
@@ -36,6 +36,8 @@
   @import 'components/airplaytogglebutton';
   @import 'components/pictureinpicturetogglebutton';
   @import 'components/spacer';
+  @import 'components/subtitlesettings/subtitlesettingsclosebutton';
```

Comment:
Please create a `'components/subtitlesettings/_subtitlesettings.scss` file that imports these 2 components, then just `@import 'components/subtitlesettings/subtitlesettings';` here. The idea of the subfolders and all that stuff is to make the subtitlesettings a component that could be a plugin or extension later on, so it is nice if as few as possible imports have to be managed.